apiService.md

```diff
@@ -103,3 +103,7 @@ API.Attribute.getAll()
 ```
 
 > This is an example of everything implemented
+
+## Automatic session validation
+
```

Comment:
## Automatic session validation

With the `APIService` generic `Send()` method, we automatically validate the user's session **before** fulfilling a send request. This way, invalid requests are never sent. In the case of an invalid request, the user logs out automatically.   <--- passive voice

Reply:
"its" removed (meant to be " 's"?) and altered last sentence (was in passive voice)